src/components/axis.js

```diff
@@ -1,4 +1,19 @@
-export default function axis() {
+export default function axis(layout, flags) {
+  const axisComponent = flags.isEnabled('CLIENT_IM_2022')
+    ? (layout.components || []).find((c) => c.key === 'axis')
+    : {};
+  const labelStyle =
+    axisComponent && axisComponent.axis && axisComponent.axis.label && axisComponent.axis.label.name
+      ? {
+          settings: {
+            labels: {
+              fontSize: axisComponent.axis.label.name.fontSize,
+              fontFamily: axisComponent.axis.label.name.fontFamily,
+              fill: axisComponent.axis.label.name.fontColor.color,
```

Comment:
This still thrown an exception:
`TypeError: can't access property "color", n.axis.label.name.fontColor is undefined`